Base/Logger/Logger.Test/FileLoggerTests.cpp

```diff
@@ -126,3 +127,21 @@ TEST_CASE("Special methods for improved logging")
     REQUIRE_NOTHROW(testLogger.info("Test info message\n"));
     REQUIRE_NOTHROW(testLogger.warning("Test warning message\n"));
 }
+
+TEST_CASE("Logging and throw exception method")
+{
+    auto& testLogger              = FileLogger::getInstance();
+    const char* filename          = "FileLoggerTest.cpp";
+    
+    std::exception* someException = new std::exception("Something wrong in std::exception");
```

Comment:
About your problem: replace the 136th line on " std::exception* someException = new std::exception(std::runtime_error("Something wrong in std::exception")); ".
I test this line in online GNU debugger and this problem is there too. My suggestion fix the problem in online debugger and should be fix checks on Unix

Reply:
I chose another way.
What does anyone else think about it?
#ifdef _WIN32
or
new std::exception(std::runtime_error(""))
Which way is better?

Reply:
Not sure that generating an exception objects is a good idea

Reply:
Can we close this subject?